Week 2 - Arrays/scrabble/scrabble.c

```diff
@@ -0,0 +1,72 @@
+#include <cs50.h>
+#include <ctype.h>
+#include <stdio.h>
+
+int getIndex(char key);
+int getValue(char key);
+int countPoints(string word);
+
+int SIZE = 26;
```

Comment:
Why do we need this variable? If it represents the length of the array then we have a convenient method to measure its length

Reply:
I also assume you already know what `const` keyword means
```suggestion
const int SIZE = 26;
```